index.html

```diff
@@ -869,34 +854,17 @@ <h4>
     A few NFC user scenarios have been enumerated
     <a href="http://www.w3.org/2009/dap/wiki/Near_field_communications_%28NFC%29#Use_cases_submitted_to_DAP_mailing_list">
     here</a> and in the
-    <a href="https://w3c.github.io/web-nfc/use-cases.html">Web NFC Use Cases</a>
-    document. The basic NFC interactions are the following.
+    <a href="/archive/use-cases.html">Web NFC Use Cases</a>
```

Comment:
I wonder if we should somehow highlight this better - graphics?

Reply:
sure

Reply:
It was archived because we moved all content in the spec. However, now we moved the content back and it might make sense to move back the use cases doc to the root (informal and showing a roadmap to further development).

Reply:
I moved back the use-cases doc to the root.